receiver/azureeventhubreceiver/README.md

```diff
@@ -0,0 +1,39 @@
+# Azure Event Hub Receiver
+
+| Status                   |                  |
+| ------------------------ |------------------|
+| Stability                | [alpha]          |
```

Comment:
This should be [`in development`](https://github.com/open-telemetry/opentelemetry-collector#in-development).

Reply:
thanks, fixed